problems/1.types+operators.js

```diff
@@ -2,15 +2,19 @@
 * @param {any} val - The input value
 * @returns {boolean} - Is val a number
 */
-function isNumber() {
-
+function isNumber (val = 15) {
+    if(val => 1)
+      return "val is a number";
 }
 
 /** Takes in a value and returns whether it is truthy
 * @param {any} val - The input value
 * @returns {boolean} - Is val truthy
 */
-function isTruthy() {
+function isTruthy(num) {
+    let i = 0
+    if(i => 0)
+    return i;
```

Comment:
This function will always return 0. Don't set i. YOu're being given an argument (which you're calling num) YOu have to check for truthyness. Remember the only falsey values in JS are 0, "",  null, undefined, NaN, false.